examples/low_level_methods/run_low_level_methods.py

```diff
@@ -117,6 +118,9 @@
 
     print(f"\n  certificates: {firewall.get_certificates()}")
 
+    print(f"\n  dynamic schedules: {firewall.get_update_schedules()}")
+    pprint(firewall.get_update_schedules())
```

Comment:
Why do we have the `firewall.get_update_schedules()` call twice?

Reply:
it's by mistake